akka-http2-support/src/main/scala/akka/http/impl/engine/http2/framing/Http2FrameParsing.scala

```diff
@@ -7,21 +7,43 @@ package framing
 
 import scala.collection.immutable
 
-import akka.stream.Attributes
```

Comment:
To be clear, I don't think we need any changes in this file to make any of the tests pass.